tailwind.config.mjs

```diff
@@ -1,8 +1,15 @@
 /** @type {import('tailwindcss').Config} */
 export default {
-	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
-	theme: {
-		extend: {},
-	},
-	plugins: [],
-}
+  content: ["./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}"],
+  theme: {
+    extend: {},
+    container: {
+      center: true,
+      padding: "1.5rem",
+      screens: {
+        lg: "1200px",
```

Comment:
I'd say this is `xl`, lg should be `1024px`

Reply:
Changed